packages/devtools_extensions/README.md

```diff
@@ -182,4 +190,47 @@ source code. Once you have done this, run `pub get`, and run the application.
 4. Connect your test app to DevTools if it is not connected already, and you should see a tab
 in the DevTools app bar for your extension. The enabled or disabled state of your extension is
 managed by DevTools, which is exposed from an "Extensions" menu in DevTools, available from the
-action buttons in the upper right corner of the screen.
\ No newline at end of file
+action buttons in the upper right corner of the screen.
+
+## Publish your package with a DevTools extension
+
+In order for a package to provide a DevTools extension to its users, it must be published with the
+expected content in the `your_package/extension/devtools/` directory (see the
+[setup instructions](#setup-your-package-to-provide-a-DevTools-extension) above).
+
+1. Ensure the `extension/devtools/config.yaml` file exists and is configured per the
+[specifications above](#setup-your-package-to-provide-a-DevTools-extension).
+2. Use the `build_and_copy` command provided by `package:devtools_extensions` to build
+your extension and copy the output to the `extension/devtools` directory:
+```sh
+cd your_extension_web_app &&
+flutter pub get &&
+dart run devtools_extensions build_and_copy \
+  --source=. \
+  --dest=path/to/your_pub_package/extension/devtools 
+```
+
+### What if I don't want the `extension/devtools/build/` contents checked into source control?
+
+As a package author, the content that you check into your git repository is completely up to you.
+If you want the contents of `extension/devtools/build/` to be git ignored, then you'll just need
+to ensure that the extension web app is always built and included in `extension/devtools/build/`
+when you publish your package.
+
+Consider adding a tool script to your repo that looks something like this:
+
+**publish.sh**
```

Comment:
nit: do we need to make this platform agnostic?

Reply:
Since this is just a suggestion for the package author, and since it is pretty simple, I think it's okay to leave as a bash script in the readme